tests/unit/dogstatsd/test_statsd.py

```diff
@@ -113,7 +113,7 @@ def test_tags_and_samples(self):
         def test_tags_and_samples(self):
             for i in range(100):
                 self.statsd.gauge('gst', 23, tags=["sampled"], sample_rate=0.9)
-            t.assert_equal('gst:23|g|@0.9|#sampled')
+            t.assert_equal('gst:23|g|@0.9|#sampled,bar:baz,foo')
```

Comment:
I don't think this test will pass, as not part of `TestDogStatsdConstantTags` class.